archive/demos/shiny-drum-machine-audio.js

```diff
@@ -0,0 +1,307 @@
+/* eslint require-jsdoc: "off" */
+
+import {INSTRUMENTS, freeze, clone} from './shiny-drum-machine-data.js';
+
+const AudioContext = window.AudioContext ?? window.webkitAudioContext;
+
+const context = new AudioContext();
+
+const LOOP_LENGTH = 16;
+const BEATS_PER_FULL_NOTE = 4;
+const VOLUMES = freeze([0, 0.3, 1]);
+
+
+async function fetchAudio(url) {
+  const response = await fetch(new Request(url));
```

Comment:
Is using a "Request" better than a plain url?

Reply:
Simplified and removed Request.